src/messages.rs

```diff
@@ -100,6 +100,7 @@ impl ChildComplete {
 #[derive(Debug, Clone, Serialize, Deserialize)]
 pub struct ChildError {
     pub error: String,
+    pub traceback: Option<String>,
```

Comment:
This is the only material change to the logic of this PR but we take the opportunity to add more tests to simulate python child return behavior.